engines/director/detection_tables.h

```diff
@@ -1364,7 +1364,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 //
 //////////////////////////////////////////////////
 
-	MACGAME1_l("alice", "", "Alice", "e54ec74aeb4355b0acd46320327c1bed", 274274, Common::JA_JPN, 200),
+	MACGAME1_l("alice", "", "Alice", "e54ec74aeb4355b0acd46320327c1bed", 271740, Common::JA_JPN, 200),
+	MACGAME1_l("alice", "Rerelease", "Alice", "e54ec74aeb4355b0acd46320327c1bed", 274274, Common::JA_JPN, 200),
```

Comment:
Is there a game version number for this one - either in the readme or somewhere in the binary?

Reply:
I can't find a version in the binary or the README, and the version field in macOS is just holding the unchanged copyright info. I've changed "rerelease" to "digipak" though.